cmd/kubeadm/app/phases/upgrade/compute.go

```diff
@@ -267,7 +268,7 @@ func GetAvailableUpgrades(versionGetterImpl VersionGetter, experimentalUpgradesA
 	}
 
 	// Add a newline in the end of this output to leave some space to the next output section
-	fmt.Println("")
+	printer.Printf("\n")
```

Comment:
```suggestion
	printer.Prinln("")
```

maybe we just have to keep these new lines between sections.